internal/drivers/nomad/tart_virtualizer.go

```diff
@@ -0,0 +1,255 @@
+package nomad
+
+import (
+	"encoding/base64"
+	"fmt"
+	cf "github.com/drone-runners/drone-runner-aws/command/config"
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/hashicorp/nomad/api"
+	"strconv"
+	"time"
+)
+
+type TartVirtualizer struct{}
```

Comment:
Let's just call it linux/mac virtualizer since it has non-tart related things like creating init jobs and returning machine frequencies/account IDs

Reply:
done